tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/mapper/ProgrammeMembershipMapper.java

```diff
@@ -203,17 +222,8 @@ private List<CurriculumMembershipDTO> programmeMembershipToCurriculumMembershipD
         = programmeMembership.getCurriculumMemberships();
 
     curriculumMemberships.forEach(cm -> {
-      CurriculumMembershipDTO cmDto = new CurriculumMembershipDTO();
-
-      cmDto.setId(cm.getId());
-      cmDto.setIntrepidId(cm.getIntrepidId());
-      cmDto.setCurriculumStartDate(cm.getCurriculumStartDate());
-      cmDto.setCurriculumEndDate(cm.getCurriculumEndDate());
-      cmDto.setPeriodOfGrace(cm.getPeriodOfGrace());
-      cmDto.setCurriculumCompletionDate(cm.getCurriculumCompletionDate());
-      cmDto.setCurriculumId(cm.getCurriculumId());
-      cmDto.setAmendedDate(cm.getAmendedDate());
-
+      CurriculumMembershipDTO cmDto =
+          curriculumMembershipMapper.curriculumMembershipToCurriculumMembershipDto(cm);
       curriculumMembershipDtos.add(cmDto);
```

Comment:
It looks like this could be simplified even further using the syntax you've used elsewhere:
```java
.stream()
.map(curriculumMembershipMapper::curriculumMembershipToCurriculumMembershipDto)
.collect()
```